vfio/src/vfio_device.rs

```diff
@@ -0,0 +1,594 @@
+// Copyright © 2019 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+//
+use byteorder::{ByteOrder, LittleEndian};
+use kvm_ioctls::*;
+use std::ffi::CString;
+use std::fmt;
+use std::fs::{File, OpenOptions};
+use std::io;
+use std::mem;
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+use std::mem::size_of;
```

Comment:
Is the thinking here to ensure that size_of is only used directly in places where the arch is x86 or x86_64 as a assist for the person writing these functions? It seems a little odd to me.

Reply:
Fixed.